app/js/actions/pastOrders.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import { LOAD_PAST_ORDERS_SUCCESS, LOAD_PAST_ORDERS_FAILURE, LOADING } from './actionTypes';
+import networkError from './networkError';
+import loading from './loading';
+
+const contextPath = window.location.href.split('/')[3];
+const apiBaseUrl = `/${contextPath}/ws/rest/v1`;
+
+export const getPastOrdersSuccess = pastOrders => ({
+  type: LOAD_PAST_ORDERS_SUCCESS,
+  pastOrders,
+});
+
+export const getPastOrdersFailure = error => ({
+  type: LOAD_PAST_ORDERS_FAILURE,
+  error,
+});
+
+export const getPastOrders = (patientUuid, careSetting) => (dispatch) => {
+  dispatch(loading(true));
+  return axios.get(`${apiBaseUrl}/order?careSetting=${careSetting}&patient=${patientUuid}&status=inactive&t=drugorder&v=full`)
+    .then((response) => {
+      dispatch(loading(false));
+      dispatch(getPastOrdersSuccess(response.data.results));
+    })
+    .catch((err) => {
+      if (!err.response) {
+        dispatch(loading(false));
+        dispatch(networkError('Network error occurred'));
```

Comment:
Is this only caused by network errors?

Reply:
err.response is undefined whenever there is a network error..https://github.com/axios/axios/issues/383#issuecomment-316501886